pulser-core/pulser/backends.py

```diff
@@ -39,13 +39,20 @@
     from pulser.backend import QPUBackend as QPUBackend
     from pulser.backend.abc import Backend
     from pulser_simulation import QutipBackendV2 as QutipBackendV2
+    from pulser_pasqal import EmuMPSBackend as EmuMPSBackend
+    from emu_mps import MPSBackend as MPSBackend
+    from emu_sv import SVBackend as SVBackend
```

Comment:
Let's add `EmuFreeBackend` here too and ignore the `mypy` errors when the package is not installed
```suggestion
    from pulser_pasqal import EmuFreeBackend as EmuFreeBackend    # type: ignore[import]
    from pulser_pasqal import EmuMPSBackend as EmuMPSBackend  # type: ignore[import]
    from emu_mps import MPSBackend as MPSBackend  # type: ignore[import]
    from emu_sv import SVBackend as SVBackend  # type: ignore[import]
```
Hopefully this doesn't go over the max line width...
Also, you'll have to run `isort .` to sort the imports